elements/pl-integer-input/pl-integer-input.py

```diff
@@ -112,7 +126,7 @@ def render(element_html, data):
             a_sub = pl.from_json(a_sub)
 
             html_params['suffix'] = suffix
-            html_params['a_sub'] = '{:d}'.format(a_sub)
+            html_params['a_sub'] = numpy.base_repr(a_sub, base=base)
```

Comment:
Nit: it doesn't really matter and I'm not requesting a change, but as a relatively inexperienced Python dev I'm curious about when one would prefer to use keyword arguments as you've done here vs. positional arguments like you use elsewhere with the base argument to `get_integer_attrib`.

Reply:
I personally like to add keyword arguments when the function receives multiple arguments and I want to make it clear which one I'm assigning a value to. In any case I removed it for consistency with the other cases.

Reply:
What does it mean for a number to be base-0? I've never encountered that idea before - how would that work?